lib/rules/function-whitespace-after/index.js

```diff
@@ -103,7 +103,7 @@ const rule = function(expectation, options, context) {
 
 			if (expectation === 'always') {
 				applyFix = (index) => {
-					fixed += value.slice(lastIndex, index) + ' ';
+					fixed += `${value.slice(lastIndex, index)} `;
```

Comment:
I think it would be better to revert this change, as it would made code more explicit about change what is happening.

Reply:
@hudochenkov the idea is to make this automatic and enforceable with an ESLint rule.

Reply:
Linters are for help, and not to blindly follow. If something seems better against linter, we could disable rule for a specific place.

Reply:
Yeah, but like I said, I want to first make sure the rule is enforced.

Reply:
That being said, feel free to make any changes you want, and when we enforce the rule we revisit the changes.